pages/index.js

```diff
@@ -80,4 +109,8 @@ function Home(props) {
   )
 }
 
+function previousYear() {
```

Comment:
no need write function separately. you can use the `useState` for this.

Reply:
`useState` to set default value ?  i need previousYear in the loop to list years as well so i made a function for it.